### PR TITLE
{devel} [GNU/4.9.3-2.25] likwid 4.0.1 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
@boegel please review

### CODE REVIEW

File: easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Ward Poelmans <ward.poelmans@ugent.be>
+diff -ur likwid-likwid-4.0.1.orig/config.mk likwid-likwid-4.0.1/config.mk
+--- likwid-likwid-4.0.1.orig/config.mk	2015-07-23 16:55:04.000000000 +0200
++++ likwid-likwid-4.0.1/config.mk	2015-10-28 13:59:52.753519484 +0100
+@@ -9,7 +9,7 @@
+ COLOR = BLUE#NO SPACE
+ 
+ # Path were to install likwid
+-PREFIX = /usr/local#NO SPACE
++#PREFIX = /usr/local#NO SPACE
```
Comment (boegel) [inline easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch]: is this really needed, doesn't this get redefined via `installopts`?

File: easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ # Versioning Information
+ VERSION = 4
+diff -ur likwid-likwid-4.0.1.orig/ext/lua/src/liolib.c likwid-likwid-4.0.1/ext/lua/src/liolib.c
+--- likwid-likwid-4.0.1.orig/ext/lua/src/liolib.c	2015-07-23 16:55:04.000000000 +0200
++++ likwid-likwid-4.0.1/ext/lua/src/liolib.c	2015-10-28 14:00:12.163071581 +0100
+@@ -19,6 +19,7 @@
+ #include <stdio.h>
+ #include <stdlib.h>
+ #include <string.h>
++#include <sys/types.h>
```
Comment (boegel) [inline easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch]: they ship their own Lua? :(:(:(

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/632d1b2d39b9730422b3 for a full test report.

File: easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch (hunk 2 above)
Comment (wpoely86) [inline easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch]: yes and hwloc too

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d8c13ae3fcd1e96dc795 for a full test report.

File: easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/l/likwid/likwid-4.0.1-config-and-lua.patch]: not tested that. It might but in this way, we're 100% sure.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1dcefde2141be4fcaa73 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b2600d73e7c599715e0 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/90691407de3b3a56851b for a full test report.

**Comment by boegel** [comment]
Thanks @wpoely86!

**Pull request merged**